dcos/marathon.py

```diff
@@ -22,10 +22,9 @@ def create_client(config=None):
         config = util.get_config()
 
     marathon_url = _get_marathon_url(config)
-    timeout = config.get('core.timeout', http.DEFAULT_TIMEOUT)
 
     logger.info('Creating marathon client with: %r', marathon_url)
-    return Client(marathon_url, timeout)
+    return Client(marathon_url)
```

Comment:
Third-party command developers write commands using the `dcos` library, and not the `dcos-cli` library, which means `dcos` has to know about CLI functionality like e.g. DCOS_CONFIG.  If it didn't, those authors wouldn't be able to write commands that use that functionality.